disperser/cmd/batcher/flags/flags.go

```diff
@@ -177,6 +177,13 @@ var (
 		EnvVar:   common.PrefixEnvVar(envVarPrefix, "MAX_BLOBS_TO_FETCH_FROM_STORE"),
 		Value:    100,
 	}
+	FinalizationBlockDelayFlag = cli.UintFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "finalization-block-delay"),
+		Usage:    "The block delay to use for pulling operator state in order to ensure the state is finalized",
+		Required: false,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "FINALIZATION_BLOCK_DELAY"),
+		Value:    75,
```

Comment:
Do we need to wait for finalization? What's the downside of using a non-final reference block? 
My understanding of the reason why we may want to use an older reference block is to prevent requesting newer blocks than what indexer has indexed. For this purpose, 5~10 blocks seems sufficient. 
One downside of having this number too big might a UX concern where operator state is applied that much later. wdyt?

Reply:
The advantage is that the OperatorState which we use is unlikely to change due to a reorg. Using 15 minutes here makes it very unlikely that we'll ever fail a batch due to a reorg as we were seeing on goerli. 

I'm not really sure that a period of 15 minutes has any important UX drawbacks. We are mostly moving in the direction of decreasing the frequency of state updates (AVSSync takes us to stake updates once per week and we may do something similar with operator registrations), so I think 15 minutes here doesn't really pose a big issue. 

I may be missing something though?

Reply:
Agreed on the minimal impact on UX and the advantage of this approach.
UX drawback was the only thing i can think of, but my concern is if there is anything else we might be missing here because we've only tested under the scenario where reference block was very recent, and we haven't seen any issues with operator state being reorg'd.
This is probably fine, but we should definitely test this in the wild for some time.

Reply:
> and we haven't seen any issues with operator state being reorg'd.

Haven't we? I think the assumption has been that the reorg issues we've been seeing on testnet were exactly this. 

But yeah, let's see if it causes any unexpected issues. I can't think if any right now.

Reply:
Another part is the operator experience where they will be held longer for dispersal after opt-out.
Overall I think it's an improvement for robustness.

Reply:
according to this twitter thread, it looks like on average 20 reorg per day. So in the worst case, need to wait 20 blocks.

Reply:
Why does 20 reorgs / day mean we need to way 20 blocks?